zkevm-circuits/src/state_circuit/test.rs

```diff
@@ -77,7 +87,7 @@ fn test_state_circuit_ok(
 fn degree() {
     let mut meta = ConstraintSystem::<Fr>::default();
     StateCircuit::configure(&mut meta);
-    assert_eq!(meta.degree(), 18);
+    assert_eq!(meta.degree(), 16);
```

Comment:
:partying_face:

Reply:
it gets better: https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/566/files#diff-854df13b82f490882b640353edbb314afff39d5d02b34c0f0d2ddeea2aab5712